app/controllers/show-project.js

```diff
@@ -29,4 +35,23 @@ export default class ShowProjectController extends Controller {
       duration: 0,
     });
   }
+
+  @action
+  handleShareOpen() {
+    this.set('shareClosed', false);
+  }
+
+  @action
+  handleShareClose() {
+    this.set('shareClosed', true);
+    this.set('copySuccess', false);
+  }
+
+  @action
+  handleShareSuccess() {
+    this.set('copySuccess', true);
+    run.later(() => {
+      this.set('copySuccess', false);
```

Comment:
FYI, testing this will random fail for the same reasons we saw earlier which is that async `set`ing on objects is unsafe because they could be destroyed once the callback happens.